marketplace-builder/src/bin/marketplace-builder.rs

```diff
@@ -0,0 +1,205 @@
+use std::{num::NonZeroUsize, path::PathBuf, str::FromStr, time::Duration};
+
+use async_compatibility_layer::logging::{setup_backtrace, setup_logging};
+use clap::Parser;
+use cld::ClDuration;
+use espresso_types::{eth_signature_key::EthKeyPair, FeeAmount, NamespaceId, SeqTypes};
+use hotshot::traits::ValidatedState;
+use hotshot_types::{data::ViewNumber, traits::node_implementation::ConsensusTime};
+use marketplace_builder::{
+    builder::{build_instance_state, BuilderConfig},
+    hooks::BidConfig,
+};
+use marketplace_builder_core::testing::basic_test::NodeType;
+use sequencer::{Genesis, L1Params};
+use snafu::Snafu;
+use url::Url;
+use vbs::version::StaticVersionType;
+
+#[derive(Parser, Clone, Debug)]
+struct NonPermissionedBuilderOptions {
+    /// Whether this is a reserve builder.
+    ///
+    /// If not, it's a fallback buidler that only builds for unregistered rollups.
+    #[clap(short, long, env = "ESPRESSO_MARKETPLACE_BUILDER_IS_RESERVE")]
+    is_reserve: bool,
+
+    /// URL of hotshot events API running on Espresso Sequencer DA committee node
+    /// The builder will subscribe to this server to receive hotshot events
+    #[clap(
+        long,
+        env = "ESPRESSO_SEQUENCER_HOTSHOT_EVENT_STREAMING_API_URL",
+        default_value = "http://localhost:8081"
```

Comment:
why is the default port 8081? Shouldn't we keep it the same as in .env (22001)

Reply:
Makes sense, it's just copypasta from legacy builder. I'll change both.